i18n.json

```diff
@@ -33,33 +31,46 @@
   { "term": "AUTHORIZATION", "wAR": "ترخيص", "wEN": "authorization" },
   { "term": "BASIC", "wAR": "أساسي", "wEN": "basic" },
   { "term": "BIND", "wAR": "ربط", "wEN": "bind" },
+  { 
+    "term": "BLOCKING_CALL", 
+    "wAR": ["استدعاء مؤجل", "استدعاء محجوب", "استدعاء غير متزامن"], 
+    "wEN": "blocking call" 
```

Comment:
```suggestion
    "wEN": "blocking" 
```

Reply:
leave it as it is for the context purpose